templates/files/empty-server/server/middleware.development.json

```diff
@@ -1,10 +0,0 @@
-{
-  "final:after": {
```

Comment:
I havent yet added the capability to read middleware.development.json. I will add the middleware.development.json in a future PR.